docs/en/02_Developer_Guides/05_Extending/00_Modules.md

```diff
@@ -101,6 +101,52 @@ operate in other structures, paths to static files such as CSS or JavaScript won
 See the [How to Publish a SilverStripe Module](how_tos/publish_a_module) for details on how to publish your SilverStripe
 modules with the community
 
+## Module Standard
+
+The SilverStripe module standard defines a set of conventions that high-quality SilverStripe modules should follow. It’s a bit like PSR for SilverStripe CMS. Suggested improvements can be raised as pull requests.
+
+### Coding Guidelines
+
+ * Declaration of level of support is provided for each module (either via README.md or composer) including the below.
+   * Level of support provided.
+   * Supporting user(s) and/or organisation(s).
+ * Complies to a well defined module directory structure and coding standards:
+   * templates (for ss templates)
+   * code (for php files)
+   * tests (for php test files) and
+   * _config (for yml config)
+ * The module is a Composer package.
+ * All Composer dependencies are bound to a single major release (e.g. ~3.1 not >=3.1).
+ * There is a level of test coverage.
```

Comment:
Capitalisation is a bit weird in this doc - e.g. `Level` should be lowercase here.